kubernetes.tf

```diff
@@ -12,7 +12,7 @@ resource "null_resource" "aws_vpc_cni" {
     environment = {
       KUBECONFIG = module.eks.kubeconfig_filename
     }
-    command = "kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v${var.cluster_aws_cni_version}/aws-k8s-cni.yaml"
+    command = "kubectl --token=${data.aws_eks_cluster_auth.cluster.token} apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v${var.cluster_aws_cni_version}/aws-k8s-cni.yaml"
```

Comment:
It's not necessary to install this manifest any longer btw, so you might as well remove it :)

Reply:
Awesome, it comes installed by default then?